recipe/conda_build_config.yaml

```diff
@@ -13,3 +13,4 @@ channel_priority:
 # Limit to R 4.3 until TileDB-SOMA-R is tested with R 4.4 on macOS
 r_base:
   - 4.3
+  - 4.4
```

Comment:
We don't want the R 4.4 build on the main branch. We know that osx-64 and osx-arm64 are failing with R 4.4 (https://github.com/TileDB-Inc/tiledbsoma-feedstock/issues/279). We are building for R 4.4 on the py39cloud branch only, which I think should be sufficient for the immediate needs of our customers

Reply:
Thanks!